erts/test/erlc_SUITE.erl

```diff
@@ -432,6 +433,21 @@ make_dep_options(Config) ->
     false = exists(BeamFileName),
     ok.
 
+compile_options(Config) ->
+    {SrcDir, _OutDir, Cmd} = get_cmd(Config),
+
+    FilenameMissingOK = filename:join(SrcDir, "erlc_test_warn_missing_spec_ok.erl"),
+    run(Config, Cmd, FilenameMissingOK, [], true),
```

Comment:
`true` is obviously wrong, here. I'm still trying to get tests running locally (for a single batch).

Reply:
These are now fixed according to test expectation.